src/QsCompiler/LanguageServer/LanguageServer.cs

```diff
@@ -126,6 +126,25 @@ public void Dispose()
         internal Task PublishDiagnosticsAsync(PublishDiagnosticParams diagnostics) =>
             this.NotifyClientAsync(Methods.TextDocumentPublishDiagnosticsName, diagnostics);
 
+        internal void CheckDotNetSDKVersion()
```

Comment:
Nitpick, but the naming convention is for acronyms longer than 2 letters to be cased as if they were a normal word, so `SDK` should be `Sdk`.

Reply:
Updated to "Sdk"